metals/src/main/scala/scala/meta/internal/metals/FileWatcher.scala

```diff
@@ -111,6 +111,7 @@ final class FileWatcher(
     // Watch the source directories for "goto definition" index.
     buildTargets.sourceRoots.foreach(watch(_, isSource = true))
     buildTargets.sourceItems.foreach(watch(_, isSource = true))
+    // TODO - do java target have to be handled as well?
```

Comment:
Once we introduce semanticdb plugin for java then we would need to handle it too.